packages/crypto-wallet-core/src/derivation/paths.ts

```diff
@@ -27,6 +27,11 @@ export const Paths = {
     livenet: "m/44'/2'/",
     testnet: "m/44'/2'/"
   },
+  MATIC: {
+    mainnet: "m/44'/966'/",
```

Comment:
woah, didn't realize matic and other eth forks have their own derivation path, since metamask lets you switch networks.

Does the BWS side let you use an ethereum wallet on the matic network?

Reply:
I haven't tested this, but I think you'd have to specify an ETH derivation path when importing from the wallet (from metamask, for example), but otherwise I don't see why that wouldn't be possible.

Reply:
I wouldn't be surprised if there is a unique index on a pubkey, though. Something to check here